standalone/src/test/java/io/pravega/local/InProcPravegaClusterTest.java

```diff
@@ -52,6 +52,11 @@ public void setUp() throws Exception {
                                            .restServerPort(TestUtils.getAvailableListenPort())
                                            .enableAuth(authEnabled)
                                            .enableTls(tlsEnabled)
+                                           .certFile("../config/cert.pem")
+                                           .keyFile("../config/key.pem")
+                                           .passwdFile("../config/passwd")
+                                           .userName("admin")
+                                           .passwd("1111_aaaa")
```

Comment:
These lines assume that the working directory of the test is the standalone directory, is that OK?   Is the gradle project configured to that effect?

Reply:
Yes. The tests work with gradle.